src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -530,7 +530,7 @@ void MacroAssembler::resolve_jobject(Register value, Register thread, Register t
 
 void MacroAssembler::stop(const char* msg) {
   address ip = pc();
-  pusha();
+  push_reg(0xffffffff, sp);
```

Comment:
Good catch, Xiaolin. But I plan to follow JDK-8245986 to use a specific instruction which will raise a SIGILL according to riscv spec. The patch will remove all these push/pop(s).

Reply:
Thank you for the explanation, Yadong. That would be better if you have future plans for this part, and I feel fine to retract this one as well because it might be not so important.